web/src/upload.js

```diff
@@ -196,6 +196,8 @@ export async function upload_files(uppy, config, files) {
             break;
         }
     }
+    get_item("send_button", config).prop("disabled", true);
+    get_item("send_button", config).css("opacity", 0.6);
```

Comment:
We shouldn't disable using CSS, but actually disable the button. Just changing the opacity won't prevent a click on the send button from sending the message.

Possibly the way to fix this is to have some global set of current reasons for disabling the send button, and disable the button only when the last reason is removed.